marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/BuildConstraints.hs

```diff
@@ -423,6 +421,9 @@ buildApplyInputsConstraintsV1 systemStart eraHistory marloweOutput invalidBefore
 
 -- | Creates a set of Tx constraints that are used to build a transaction that
 -- withdraws payments from a payout validator.
+
+execWriterT :: WriterT (TxConstraints v0) (Either (ApplyInputsError 'V1)) () -> Either (ApplyInputsError 'V1) (TxConstraints 'V1)
+execWriterT = error "not implemented"
```

Comment:
Was this left here by mistake?
I think [execWriterT :: (Monad m, Monoid w) => WriterT w m a -> m w](https://hackage.haskell.org/package/transformers-0.6.0.4/docs/Control-Monad-Trans-Writer-CPS.html#v:execWriterT) should work the way I'd expect this procedure to work. ie

```haskell
-- Evaluates to Left "boo"
execWriterT do
  tell ["datum1", "datum2"]
  lift $ Left "nay"
```
```haskell
-- Evaluates to Right ["datum1", "datum2"]
execWriterT do
  tell ["datum1", "datum2"]
  lift $ Right "yay"
```

Reply:
This was indeed created by mistake and left by mistake (I selected the `Define function execWriterT :: ...` code action instead of the `import Control.Monad.Writer (execWriterT)` one